docs/installing-solidity.rst

```diff
@@ -210,12 +210,30 @@ You will need to install the following dependencies for Windows builds of Solidi
 +------------------------------+-------------------------------------------------------+
 | `CMake`_                     | Cross-platform build file generator.                  |
```

Comment:
The `|` has to be moved to the right now, I think. Otherwise, the table does not render correctly.